components/post_markdown/index.js

```diff
@@ -1,6 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {connect} from 'react-redux';
+
 import PostMarkdown from './post_markdown';
 
-export default PostMarkdown;
+function mapStateToProps(state) {
+    return {
+        pluginHooks: state.plugins.hooks.MessageWillFormat,
```

Comment:
You'll need to switch this to `state.plugins.components.MessageWillFormat` now as well